src/components/SigningModal/TransactionJourney.js

```diff
@@ -200,40 +217,32 @@ function JourneyItem({
       <div
         css={`
           position: relative;
-          margin-right: ${2.5 * GU}px;
+          margin-right: ${2 * GU}px;
         `}
       >
         {!isLastItem && (
           <ProgressLine status={status} pulse={pulse} themeColor={themeColor} />
         )}
+
         <div
           css={`
             position: relative;
-            display: flex;
-
             overflow: hidden;
-
-            width: ${5 * GU}px;
-            height: ${5 * GU}px;
-
-            background-color: ${theme.badge};
-            border-radius: ${RADIUS}px;
             z-index: 1;
-
-            &::after {
-              content: '';
-
-              background-image: url(${icon});
-              flex: 1;
-
-              ${status === NEXT &&
-                `filter: grayscale(97%);
-                mix-blend-mode: hard-light;
-                opacity: 0.8;
-              `}
-            }
+            border-radius: ${RADIUS}px;
+            background-color: ${theme.surfaceUnder};
           `}
-        />
+        >
+          <div
+            css={`
+              ${disabled && disabledIconStyle};
+              ${disabledLight && disabledLightIconStyle};
+              ${disabledDark && disabledDarkIconStyle};
```

Comment:
Similar to above, I'm wondering if there's an opportunity to simplify this so it's a bit more clear which styles are being applied when.

Perhaps something like:

```js
${if (disabled) {
  return `
    ${disabledIconStyle};
    ${appearance === 'light'
      ? disabledLightIconStyle
      : appearance === 'dark'
      ? disabledDarkIconStyle
      : ''
  `
}}
```

Alternatively, I think we could inject `disabledIconStyle` into the other styles, to simplify the condition:

```js
const disabledLightIconStyle = css`
  {disabledIconStyle};
  opacity: 0.4;
`
```